docs: remove v0 mentions, minor improvements (#371)
VRSEN/agency-swarm: docs/references/api.mdx

    """
    [DEPRECATED] Use get_response_stream instead.

>     This method is not supported in the current architecture due to structural
>     changes. Use get_response_stream() for streaming functionality.

Reviewer: ```suggestion
    This method is deprecated.
    Use get_response_stream() for streaming functionality.
```
